dev/CommandBarFlyout/CommandBarFlyout.xaml

```diff
@@ -258,21 +258,20 @@
                                 </VisualState>
                             </VisualStateGroup>
                         </VisualStateManager.VisualStateGroups>
-                        <Grid x:Name="OuterContentRoot">
+                        <Grid x:Name="OuterContentRoot"
+                            VerticalAlignment="Top"
+                            Margin="{TemplateBinding Padding}"
+                            Width="{Binding RelativeSource={RelativeSource TemplatedParent}, Path=FlyoutTemplateSettings.CurrentWidth}"
+                            Height="{TemplateBinding Height}"
+                            Background="{TemplateBinding Background}">
                             <Grid.Clip>
                                 <RectangleGeometry x:Name="OuterContentRootClip" Rect="{Binding RelativeSource={RelativeSource TemplatedParent}, Path=FlyoutTemplateSettings.ContentClipRect}">
                                     <RectangleGeometry.Transform>
                                         <TranslateTransform x:Name="OuterContentRootClipTransform" />
                                     </RectangleGeometry.Transform>
                                 </RectangleGeometry>
                             </Grid.Clip>
-                            <Grid x:Name="ContentRoot"
-                                VerticalAlignment="Top"
-                                Margin="{TemplateBinding Padding}"
-                                Width="{Binding RelativeSource={RelativeSource TemplatedParent}, Path=FlyoutTemplateSettings.CurrentWidth}"
-                                Height="{TemplateBinding Height}"
-                                Background="{TemplateBinding Background}"
-                                contract4Present:XYFocusKeyboardNavigation="Enabled">
```

Comment:
Did you intend to remove XYFocusKeyboardNavigation?

Reply:
Whoops, nope - thanks, good catch.